cmd/node/config/p2p.toml

```diff
@@ -60,5 +60,5 @@
 [Sharding]
     # Number of ptiority bits used
     PrioBits = 4
-    MaxIntraShard = 15
-    MaxCrossShard = 15
+    MaxIntraShard = 12
+    MaxCrossShard = 4
```

Comment:
Maybe min connection to meta? To have at least one?

Reply:
We can do this. The reason this is not implemented is that the metanodes will be overwhelmed with connections as every shard node will try to connect to a meta node. We can experiment with this but we'll need a larger setup as to observe the metanodes behavior. It will work for now as it is. Added a TODO to study this.